tao/tao_bridge/passes/tao_mark_for_compilation_pass.cc

```diff
@@ -1503,30 +1503,33 @@ Status MarkForCompilationPassImpl::FindCompilationCandidates() {
           continue;
         }
       }
+    }
 
-      bool to_continue = false;
-      for (auto s : absl::StrSplit(
-               GetTaoBridgeOptions()->op_type_clustering_black_list, ',')) {
-        if (s == node->type_string()) {
-          VLOG(2) << "Rejecting " << node->type_string()
-                  << " as op type black list";
-          to_continue = true;
-          break;
-        }
+    bool to_continue = false;
+    for (auto s :
+         absl::StrSplit(GetTaoBridgeOptions()->op_type_clustering_black_list,
```

Comment:
nit: make sense to skip empty splits, but just curious when will `node->type_string()` be empty?

Reply:
This could be from env var, so user can give anything, ignoring empty string to make it more robust